src/wrapper.test.js

```diff
@@ -179,220 +189,328 @@ describe('Distro initialization', () => {
         process.env.LUMIGO_DEBUG_SPANDUMP = '/dev/stdout';
       });
 
-      test('should initialize the FileSpanExporter', () => {
-        jest.isolateModules(async () => {
+      test('should initialize the FileSpanExporter', async () => {
+        await jest.isolateModulesAsync(async () => {
           const wrapper = jest.requireActual('./wrapper');
-          await wrapper.init.then(() => {
-            expect(FileSpanExporter).toHaveBeenCalledWith('/dev/stdout');
-          });
+          await wrapper.init;
+
+          expect(FileSpanExporter).toHaveBeenCalledWith('/dev/stdout');
         });
       });
     });
   });
 
   describe('outside of a computing platform for which we have detectors', () => {
+    beforeEach(() => {
+      process.env.LUMIGO_TRACER_TOKEN = LUMIGO_TRACER_TOKEN;
+      process.env.OTEL_SERVICE_NAME = 'service-1';
+      process.env.LUMIGO_REPORT_DEPENDENCIES = 'false';
+    });
+
     test('NodeTracerProvider should be given a resource with all the right attributes', async () => {
```

Comment:
test with the same name repeat 4 times, although each is in another "describe" suite, can you pleas change the test name to be more precise?

Reply:
What is the actual problem? The hierarchy matters, and that is how the failed tests are visualized: with the full hierarchy. Feel free to change them, but in a different PR.